core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/stringfunction/NowStrExpression.kt

```diff
@@ -5,12 +5,12 @@ import ch.ergon.dope.resolvable.expression.unaliased.type.toStringType
 import ch.ergon.dope.resolvable.operator.FunctionOperator
 import ch.ergon.dope.validtype.StringType
 
-class NowStrExpression(private val fmt: TypeExpression<StringType>) : TypeExpression<StringType>, FunctionOperator {
-    override fun toQueryString(): String = this.toFunctionQueryString("NOW_STR", fmt)
+class NowStrExpression(
+    private val fmt: TypeExpression<StringType>,
+) : TypeExpression<StringType>, FunctionOperator {
+    override fun toQueryString(): String = toFunctionQueryString("NOW_STR", fmt)
 }
 
-fun nowStr(fmt: TypeExpression<StringType>): NowStrExpression = NowStrExpression(fmt)
-
-fun nowStr(): NowStrExpression = nowStr("".toStringType())
+fun nowStr(fmt: TypeExpression<StringType> = "".toStringType()): NowStrExpression = NowStrExpression(fmt)
```

Comment:
Maybe add a comment referring to ticket Dope-177

Reply:
done